widgets/projection/powder/powder.py

```diff
@@ -1,8 +1,44 @@
 from powder_ui import Ui_Form
 from PyQt4.QtGui import QWidget
+from presenters.powder_projection_presenter import PowderProjectionPresenter
+from models.projection.powder.mantid_projection_calculator import MantidProjectionCalculator
+from views.powder_projection_view import PowderView
+from command import Command
 
 
-class PowderWidget(QWidget,Ui_Form):
+class PowderWidget(QWidget,Ui_Form,PowderView):
+    """This widget is not usable without a main window which implements mainview"""
     def __init__(self,*args, **kwargs):
         super(PowderWidget,self).__init__(*args, **kwargs)
         self.setupUi(self)
+        self.btnPowderCalculateProjection.clicked.connect(self._btn_clicked)
+
+    def set_main_window(self, main_window):
```

Comment:
The main window must implement mainview. Any information flowing between tabs goes via the the mainviews presenter